src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.Scanner;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private final String DEFAULT_PATH = "./duke.txt";
+    private final Path filePath;
+
+    public Storage() {
+        filePath = Paths.get(DEFAULT_PATH);
+        File file = new File(DEFAULT_PATH);
+        if (Files.notExists(this.filePath)) {
+            try {
+                System.out.println("created new file");
+                file.createNewFile();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+
+    public void writeData(List<Task> taskList) throws IOException {
+        FileWriter file = new FileWriter(DEFAULT_PATH);
+        for (Task task : taskList) {
+            String toBeSaved = "";
+            switch (task.getTaskType()) {
+                case "DEADLINE":
```

Comment:
Is the switch statement case indentation supposed be as such?

Reply:
Second this. I believe your switch-case statements in your Parser were indented as per Java coding standards.

Reply:
Yup thanks for catching this!! :)